lib/src/health.dart

```diff
@@ -161,3 +181,10 @@ List<Suggestion> _compact(
   suggestions.sort();
   return suggestions;
 }
+
+class _Suggestion {
```

Comment:
The distinction between Suggestion and _Suggestion is confusing. _SuggestionWithPercent? Can we avoid this class or make it more like a special purpose class?